packages/cicero-core/src/templatemetadata.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Util = require('./util');
+const Metadata = require('./metadata');
+
+/**
+ * Defines the metadata for a Template, including the name, version, README markdown.
+ * @class
+ * @public
+ */
+class TemplateMetadata extends Metadata {
```

Comment:
Ah, I see, you refactored TemplateMetadata into its own class, using Metadata for instances.

Reply:
TBD. @jeromesimeon ?